x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/AbstractLongScriptFieldScript.java

```diff
@@ -23,6 +23,8 @@ public AbstractLongScriptFieldScript(Map<String, Object> params, SearchLookup se
         super(params, searchLookup, ctx);
     }
 
+    public abstract void execute();
```

Comment:
Because I'm only doing one of them I have to move the old execute method decalarations into the subclasses.